src/Microsoft.SqlTools.ManagedBatchParser/Microsoft.SqlTools.ManagedBatchParser.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<!-- Targeting netstandard2.0 so that other things such as PS cmdlets can use this which need to support a wider range of machines -->
-		<TargetFramework>netstandard2.0</TargetFramework>
+		<!-- Targeting netstandard2.1 so that other things such as PS cmdlets can use this which need to support a wider range of machines -->
+		<TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
@Matteo-T FYI

Reply:
Any chance this one project can be made available for, say, net472 as well as netstandard2.0?

What is that it is making it so netstandard2.1-specific, which would essentially make it unable to interop (without taking risks) with NetFx (full framework)?

Reply:
> What is that it is making it so netstandard2.1-specific, which would essentially make it unable to interop (without taking risks) with NetFx (full framework)?

Recently DacFx modified their support for .NET Standard to 2.1, that drove this change.